### ISSUE TITLE
Full locomotion speed on single hand

### ISSUE THREAD

**Issue by IDLauncher** [Frooxius/NeosPublic#1720]
## Is your tweak request related to a problem? Please describe.
Currently, moving at top speed forward or backward requires holding that direction on both hand controllers at the same time. I find this awkward.

## Describe what would you like tweaked
Adding an option that would allow users the full range of speed on a single controller input instead of dividing it across both hands. (Personally I'd put it all on the side that also strafes.)

## Describe alternatives you've considered
I dug around in SteamVR settings to see if I could achieve something similar with remapping controls, but didn't figure out a solution.

## Additional context
Side benefit: On an index controller, it'd make it easier to move at full tilt and use jump at the same time if you've remapped it to a button instead of stick click. Index stick clicks wear down fairly quick, so remapping it is good for longevity of the stick click. Plus the analog part of the analog stick feels kind of wasted on the half speed range.

**Comment by shiftyscales** [comment]
This is something you should be able to set up on a per-locomotion module basis. There is a property called `DualInputHandling` which determines how to use the input from both joysticks. I believe setting it to `PrimaryOnly` Was supposed to restrict it to only using one stick, or you could use `ClampedSum` which would limit it to the speed as defined by the locomotion module.

However, on investigating, I'm finding that it appears this DualInputHandling property appears to be broken at the moment, the locomotion is acting the same way with both sticks regardless of what the property is set to.

I know @Frooxius is currently in the process of working out the new input binding system, and how to integrate it into the locomotion system, etc. so perhaps this is an area he could look at/consider how to handle?

**Comment by IDLauncher** [comment]
That would be good. The current setup has been a pet peeve of mine for a long time now.

I actually jumped into Neos to try your suggestion just to see what happens. I hunted for documentation on how to find that module but didn't manage to find any on it, but I figured it out in game.

I did manage to end up totally disabling my left joystick using "PrimaryOnly" which is _almost_ the opposite of the desired result, but I was mostly just fiddling to learn this opaque thing and experiment a little since you'd already said it was broken.

Thanks for taking the time to reply there. c:
